checkov/terraform/checks_infra/solvers/attribute_solvers/exists_attribute_solver.py

```diff
@@ -1,13 +1,14 @@
+from typing import List, Optional, Any, Dict
+
 from checkov.common.graph.checks_infra.enums import Operators
 from checkov.terraform.checks_infra.solvers.attribute_solvers.base_attribute_solver import BaseAttributeSolver
 
 
 class ExistsAttributeSolver(BaseAttributeSolver):
     operator = Operators.EXISTS
 
-    def __init__(self, resource_types, attribute, value):
-        super().__init__(resource_types=resource_types,
-                         attribute=attribute, value=value)
+    def __init__(self, resource_types: List[str], attribute: Optional[str], value: Any) -> None:
+        super().__init__(resource_types=resource_types, attribute=attribute, value=value)
 
-    def _get_operation(self, vertex, attribute):
+    def _get_operation(self, vertex: Dict[str, Any], attribute: Optional[str]) -> bool:
```

Comment:
@gruebel Pretty sure attribute shouldn't be Optional in this method and its inheritances too

Reply:
same here, this is related to connection solver, where `attribute` is `None`.